field_manual/alertness.toml

```
@@ -0,0 +1,20 @@
+displayName = "Alertness"
+description = """
+When you're running a mission, many actions you take will alert the enemy that there's a S.O.G recon team operating in their area.
+
+As the enemy forces become more aware of your presence, they'll start [h]actively hunting your team[/h], and [h]respond quicker to your activities[/h].
+
+This means you should be careful when doing anything loud, or that might draw attention to you. [h]Gunshots and explosions are a surefire way to compromise your mission[/h].
+
+And remember, [h]a suppressed weapon is not a silent weapon[/h].
+"""
+
+arguments = ["localize 'STR_VGM_MISSION_NAME'"]
```

I don't see the argument being used.
```suggestion
arguments = []
```